share/spice/just_delete_me/just_delete_me.js

```diff
@@ -12,19 +12,36 @@
         if (!decodedQuery || decodedQuery.length < 4) {
             return Spice.failed('just_delete_me');
         }
-
-        var exact = api_result.filter(function(item) { return item.name.toLowerCase() === decodedQuery}),
-            exact = exact.length > 0 ? [exact[0]] : 
-                    api_result.filter(function(item) { return item.domains &&
-                        ((typeof(item.domains) === "string" && item.domains.toLowerCase() === decodedQuery) ||
-                         typeof(item.domains) === "object" && item.domains.some(function(domain) {return domain.toLowerCase() === decodedQuery}) )
-                    }),
-            api_result = exact.length > 0 ? [exact[0]] : api_result.filter(function (item) {
-                        item.domains =  !item.domains ? [] : typeof(item.domains) === "string" ? [item.domains] : item.domains;
-                        if (item.name.toLowerCase().includes(decodedQuery) || item.domains.some(function(domain) {return domain.toLowerCase().includes(decodedQuery);}))
-                            return true;
-                        });
-
+        var matchPartial = function(item) {
+            return (item.name.toLowerCase().includes(this) || item.domains.some(domain => domain.includes(this)));
```

Comment:
@riqpe I just noticed you'd mixed in some ES6 syntax here -- it looks like it's causing some problems internally. Please switch this to ES5 -- we don't support ES6 yet.